packages/material-ui-lab/src/TimelineSeparator/TimelineSeparator.js

```diff
@@ -1,22 +1,63 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import { withStyles } from '@material-ui/core/styles';
+import { unstable_composeClasses as composeClasses } from '@material-ui/unstyled';
+import {
+  experimentalStyled,
+  unstable_useThemeProps as useThemeProps,
+} from '@material-ui/core/styles';
+import { getTimelineSeparatorUtilityClass } from './timelineSeparatorClasses';
 
-export const styles = () => ({
+const overridesResolver = (props, styles) => styles.root || {};
+
+const useUtilityClasses = (styleProps) => {
+  const { classes } = styleProps;
+
+  const slots = {
+    root: ['root'],
+  };
+
+  return composeClasses(slots, getTimelineSeparatorUtilityClass, classes);
+};
+
+const TimelineSeparatorRoot = experimentalStyled(
+  'div',
+  {},
+  {
+    name: 'MuiTimelineSeparator',
+    slot: 'Root',
+    overridesResolver,
+  },
+)(() => {
```

Comment:
No need for function

Reply:
@mnajdova But should it be set the same to maintain consistency or should no more be added for this case? My question is that in the previous PR it also went that way

Reply:
In the other PR there was use of the `theme`, here we don't need function,it's. Ilways better for perf not to have to evaluate the function always if there is no need for it

Reply:
@mnajdova I understand, thanks for the explanation.